scripts/ip-geolocation-maxmind.nse

```diff
@@ -31,8 +34,26 @@ author = "Gorjan Petrovski"
 license = "Same as Nmap--See https://nmap.org/book/man-legal.html"
 categories = {"discovery","external","safe"}
 
+local db_file
```

Comment:
This is insufficient to cache the database file. File-scoped local variables are reinstantiated for each script execution, so it doesn't save anything. This is what the NSE registry is for, so let's use it instead.